learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -65,8 +65,8 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 
 |         | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
-| Schemaless | ✅ | ✅ | 🔶 <br>Automatic schema detection is supported but needs to be specified | ✅ |
-| Nested field support | ✅ | ✅ | ❌ | ✅ |
+| Schemaless | ✅ | ✅ | ✅ <br>Automatic schema detection needs to be enabled | ✅ |
+| Nested field support | ✅ | ✅ | ✅ | ✅ |
```

Comment:
Launched in v0.24.0

Reply:
I'm not even sure it needs to be specified, `✅` would be enough I think

Reply:
Hey @jasonbosco I saw your thumbs up. I guess this means you're happy with a green checkmark, so I will edit the comparison accordingly